frontend/src/common/enums/enums.ts

```diff
@@ -18,6 +18,7 @@ export {
   PodcastType,
   PodcastSearchPayloadKey,
   PodcastPeriodicity,
+  PodcastLoadFilterKey,
```

Comment:
in the shared folder we have `PodcastSearchPayload`, `PodcastLoadFilterKey`
in this case we should have only one